client/maintenance.go

```diff
@@ -34,11 +34,11 @@ type LoadTask struct {
 }
 
 func (t *LoadTask) Await(ctx context.Context) error {
-	ticker := time.NewTicker(t.interval)
-	defer ticker.Stop()
+	timer := time.NewTimer(t.interval)
+	defer timer.Stop()
 	for {
 		select {
-		case <-ticker.C:
+		case <-timer.C:
 			loaded := false
 			t.client.callService(func(milvusService milvuspb.MilvusServiceClient) error {
```

Comment:
can we add an error message here if call service failed?
Don't think it's good to ignore errors.
For example, if we hit collection not exist or some unrecoverable error, we should directly return error instead of timed retry

Reply:
If an error occurs here, it has been returned directly. The retry logic here is not for the error, but for the loading progress not reaching 100%.

![截屏2024-08-19 10 17 31](https://github.com/user-attachments/assets/940784c0-2ebc-46b4-aa91-23431806f83f)

Reply:
callService will return an error it is not handled

Reply:
Sorry, I ignored it just now. This place needs to return an error directly. 

At the same time, I think that when no error occurs, but the progress does not reach 100%, it is also necessary to change to a timer instead of a ticker to try again.